PrestaShop/contextual-help-api
# Add missing Gitbook markdown parsing feature

| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | This PR adds support for parsing Gitbook's specific markdown syntax
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fix https://github.com/PrestaShop/PrestaShop/issues/29287 and related to https://github.com/PrestaShop/PrestaShop/issues/29216
| How to test?      | See below
| Possible impacts? | 

### How to test

1. In the file `admin-dev/themes/default/js/help.js`, line 28 change `https://help.prestashop-project.org` with `https://integration-help.prestashop-project.org`:
```diff
- const psDocsDomain = 'https://help.prestashop-project.org';
+ const psDocsDomain = 'https://integration-help.prestashop-project.org';
```
2. In the file `classes/controller/AdminController.php`, line 1669 change `https://help.prestashop-project.org` with `https://integration-help.prestashop-project.org`:
```diff
- $this->context->smarty->assign('help_link', 'https://help.prestashop-project.org/' . Language::getIsoById($this->context->employee->id_lang) . '/doc/'
+ $this->context->smarty->assign('help_link', 'https://integration-help.prestashop-project.org/' . Language::getIsoById($this->context->employee->id_lang) . '/doc/'
```
3. In the file `src/PrestaShopBundle/Resources/config/services/core/help.yml`, line 9 change `https://help.prestashop-project.org` with `https://integration-help.prestashop-project.org`:
```diff
- - "https://help.prestashop-project.org/"
+ - "https://integration-help.prestashop-project.org/"
```
4. Go on the BO and check that the contextual help is well displayed both in legacy and migrated controllers
